analyzer/analyzer.go

```diff
@@ -1022,6 +1026,18 @@ func (pd *ParsedData) parseBugReport(fnameA, contentsA, fnameB, contentsB string
 	return nil
 }
 
+func constructAppName(output sensorserviceutils.OutputData,
```

Comment:
Need function header. In general if you find that this file is missing function headers, please add them